tools/clang/test/taef_exec/lit.cfg

```diff
@@ -0,0 +1,68 @@
+# -*- Python -*-
+
+# Configuration file for the 'lit' test runner.
+
+import os
+import platform
+
+import lit.formats
+import lit.util
+
+# name: The name of this test suite.
+config.name = 'clang-taef-exec'
+
+# Check that the object root is known.
+te = getattr(config, 'te', None)
+if te is None:
+    # Otherwise, we haven't loaded the site specific configuration (the user is
+    # probably trying to run on a test file directly, and either the site
+    # configuration hasn't been created by the build system, or we are in an
+    # out-of-tree build situation).
+
+    # Check for 'clang_taef_exec_site_config' user parameter, and use that if available.
+    site_cfg = lit_config.params.get('clang_taef_exec_site_config', None)
+    if site_cfg and os.path.exists(site_cfg):
+        lit_config.load_config(config, site_cfg)
+        raise SystemExit
+
+# TAEF only runs on Windows
+if platform.system() != 'Windows':
+  config.unsupported = True
+else:
+  test_dll = os.path.join(config.llvm_obj_root, config.llvm_build_mode, 'bin', 'ExecHLSLTests.dll')
+
+  hlsl_data_dir = os.path.join(config.llvm_src_root, 'tools', 'clang', 'unittests', 'HLSLExec')
+
+  test_dir = os.path.join(config.llvm_obj_root, config.llvm_build_mode, 'test')
+  param_hlsl_data_dir = str.format('/p:HlslDataDir={}', hlsl_data_dir)
+
+  extra_params = ["/p:\"ExperimentalShaders=*\"", param_hlsl_data_dir]
+
+  verbose = lit_config.params.get('verbose', None)
+  if verbose == None:
+      extra_params.append('/logOutput:LowWithConsoleBuffering')
+
+  arch = getattr(config, 'taef_arch', None)
+  arch_filter = str.format("@Architecture='{}'", arch)
+  priority_filter = "@Priority<2"
+  future_filter = lit_config.params.get('future_filter', None)
+  if future_filter:
+      priority_filter = "@Priority=2"
```

Comment:
I'm not aware of any Execution tests that have a Priority of 2. they all look to be 0

Reply:
Priority=2 was intended for in-development features that wouldn't run normally on standard runtime/warp, but needed a specific configuration to run.  So you wouldn't want to run them in normal test pass.

Reply:
Sure, I don't mind it being here to much as I worry about it not being in the other runner since ther are Execution and DxilContainer tests that make use of it. When not specified with a flag, does it default to 0? Maybe that's fine? If we ever want to promote those tests to more regular testing, maybe we'll just update their priorities.

Incidentally, googletest has no notion of priorities, but you can disable tests that can be run if you explicitly allow disabled tests. So this might have to change going forward anyway